app/controllers/v1/todays_project_controller.rb

```diff
@@ -1,5 +1,7 @@
 module V1
 	class TodaysProjectController < ApiController
+    skip_before_action :authenticate_v1_user!
```

Comment:
Instead of skipping for every action you can inherit from UnauthenticatedController instead of ApiController

Reply:
nbundsa